tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/pages/self-assessment.cy.js

```diff
@@ -1,5 +1,7 @@
+import { selfAssessmentSlug } from "../../helpers/page-slugs";
+
 describe("Self-assessment page", () => {
-  const url = "/self-assessment";
+  const url = selfAssessment;
```

Comment:
Should this be `const url = selfAssessmentSlug;`?

Reply:
Good spot. I renamed it but missed it here. Cheers buddy